src/modules/apps/exec.ts

```diff
@@ -0,0 +1,53 @@
+import * as WebSocket from 'ws'
+import * as url from 'url'
+import { getAccount, getToken, getWorkspace } from '../../conf'
+import { getManifest } from '../../manifest'
+import { region } from '../../env'
+import { toMajorRange } from '../../locator'
+const EOT = '\x04'
+export default async (command: string, options) => {
+  const manifest = await getManifest()
+  const { name, vendor, version, builders } = manifest
+  const { dotnet, node, 'service-js': serviceJs } = builders
+  if (!dotnet && !node && !serviceJs) {
+    return
+  }
+  const majorRange = toMajorRange(version)
```

Comment:
I will add a `majorRange` getter on `ManifestEditor`, then it could be `manifest.majorRange`

Reply:
Added :)